src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -815,8 +815,9 @@ public sealed class HttpResponseException : HttpRequestException
         /// Initializes a new instance of the <see cref="HttpResponseException"/> class.
         /// </summary>
         /// <param name="message">Message for the exception.</param>
+        /// <param name="statusCode">Status Code from the HTTP server.</param>
         /// <param name="response">Response from the HTTP server.</param>
-        public HttpResponseException(string message, HttpResponseMessage response) : base(message)
+        public HttpResponseException(string message, HttpStatusCode statusCode, HttpResponseMessage response) : base(message, null, statusCode)
```

Comment:
It is a breaking change. We cannot change a public API. We could add new constructor but I suggest use simple assignment. See my next comment.